src/client.js

```diff
@@ -75,6 +78,9 @@ function createMethod(clientMethod, dbg, cancelCache) {
       }
 
       if (clientMethod.responseStream) {
+        const domain = createDomain();
```

Comment:
This is a semi important fix here as no domain was ever being passed in. Without it server errors like `RST_STREAM`... etc are not handled and just blow up node. With this it forwards the domain errors to the observable. Thus now it's handled.